Scripts/script-ParsePCAP.yml

```diff
@@ -0,0 +1,286 @@
+commonfields:
+  id: 26f3f224-e936-4a9d-879c-aa3edcd9862b
+  version: 30
+name: ParsePCAP
+script: |+
+  import socket
+  import itertools
+  import operator
+  import pip
+
+
+  pip.main(['-q', 'install', 'cymruwhois'])
```

Comment:
https://github.com/demisto/tools/tree/master/docker

and add docker image name here:

![image](https://cloud.githubusercontent.com/assets/6059211/25117861/85e53edc-241c-11e7-8f58-d132392baf21.png)